src/plugins/embeddable/public/lib/panel/panel_header/panel_header.tsx

```diff
@@ -174,26 +175,36 @@ export function PanelHeader({
     );
   }
 
+  const renderTitle = () => {
+    const titleComponent = showTitle ? (
+      <span className={title ? 'embPanel__titleText' : 'embPanel__placeholderTitleText'}>
+        {title || placeholderTitle}
+      </span>
+    ) : undefined;
+    return description ? (
+      <EuiToolTip
+        content={description}
+        delay="regular"
+        position="right"
+        anchorClassName="embPanel__maxWidth"
```

Comment:
In BEM naming practices, you always want to use class names that describe the element not the style. This way it's more scalable in case more styles need to be added later.

Reply:
This makes sense, have renamed this class